src/photos/components/Viewer.jsx

```diff
@@ -199,13 +202,13 @@ export class Viewer extends Component {
         x: 0,
         y: 0
       },
-      hideToolbar: true
+      hideActions: true
     }))
   }
 
   componentDidUpdate (prevProps, prevState) {
-    // if the toolbar was hidden but is now displayed, start a countdown to hide it again
-    if (prevState.hideToolbar && !this.state.hideToolbar) this.hideToolbarAfterDelay()
+    // if the viewer's actions were hidden but are now displayed, start a countdown to hide them again
+    if (prevState.hideActions && !this.state.hideActions) this.hideActionsAfterDelay()
```

Comment:
If I understand the code:

1. `prevState.hideActions` is true, ie. if **viewer's actions were hidden**
1. `this.state.hideActions` is false, ie. if **viewer's actions are now displayed**
1. `this.hideActionsAfterDelay()` is executed, ie. **start a countdown to hide them again**

So, what's the purpose of the comments since the code is explicit?

> Clean Code let us omit comments for the benefit of explicit code